rpm/kubelet.spec

```diff
@@ -1,4 +1,4 @@
-%global KUBE_VERSION 1.4.1
+%global KUBE_VERSION 1.4.3
```

Comment:
Perfect world we should have an entry in the %changelog for this and some notes on what's coming in the new package. I can draft one if you like.